src/main/webapp/js/user-page-loader.js

```diff
@@ -55,8 +69,8 @@ function fetchAboutMe(){
   fetch(url).then((response) => {
     return response.text();
   }).then((aboutMe) => {
-    const aboutMeContainer = document.getElementById('about-me-container');
-    if(aboutMe == ''){
+    const aboutMeContainer = document.getElementById('about-me-text');
+    if(/\s/.test(aboutMe)){
```

Comment:
Sorry my mistake! I've fixed it to be !(/S/.test(aboutMe)).
I'm adding this whitespace check because I find out it is possible that the input is simply '\n', then as a result, user could never select the "\n" and change it to something meaningful again.